packages/test-commerce-sdk-react/app/pages/useShopperProducts.tsx

```diff
@@ -0,0 +1,23 @@
+/*
```

Comment:
Should we use `use-shopper-products` as name instead of captialized for file name?

Reply:
Are we gonna make each page for each hook use? 🤔

Reply:
so just to set expectations, this is an internal tool. The purpose of this tool is to speed up our development, make us develop the `commerce-sdk-react` faster.

> Are we gonna make each page for each hook use? 🤔

Short answer: No!

Longer answer: if you think you will need to test / recreate the setup in the future for development and debugging purposes, sure yes please add those examples. But it is by no means a requirement, it's something that we build and it benefits us in the long run.